librz/include/rz_th.h

```diff
@@ -34,7 +34,7 @@ typedef struct rz_th_t RzThread;
 typedef struct rz_th_pool_t RzThreadPool;
 typedef struct rz_th_queue_t RzThreadQueue;
 typedef void *(*RzThreadFunction)(void *user);
-typedef void (*RzThreadIterator)(void *element, void *user);
```

Comment:
Ah ok. Yeah, I didn't get it at all :D

Better add a doxygen here I think, when the `RzThreadIterator` is supposed to return false and when true. Also to know what the parameters are.